pkg/kubelet/kubelet_pods.go

```diff
@@ -85,6 +87,8 @@ const (
 	kubeletUser = "kubelet"
 )
 
+const sysctlAllDisableIPv6 = "net/ipv6/conf/all/disable_ipv6"
```

Comment:
I'd suggest to move this constant to the utilsysctl module as it already has similar constants used by Kubelet: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/component-helpers/node/util/sysctl/sysctl.go#L27-L46

Reply:
I considered doing this, but ultimately decided to put it on the caller.

The sysctl is not quite in the same category as those in the utilsysctl module, and perhaps we should be cautious about adding constants to `k8s.io/component-helpers` unless we use it in multiple components.